ui/ducks/send/send.js

```diff
@@ -1419,6 +1441,13 @@ const slice = createSlice({
             });
             draftTransaction.status = SEND_STATUSES.INVALID;
             break;
+          case draftTransaction.amount.value === '0x0' ||
+            draftTransaction.amount.value === '0x0':
```

Comment:
The two checks here on this || statement seem to be identical, is this intentional? IF so we probably only need one.

Reply:
My mistake, it should not be identical, it should with '0x0' and '0x00', i will change that